src/core/ui/manager.tsx

```diff
@@ -221,7 +221,7 @@ const ComponentManagerComponent = (props: ComponentManagerComponentProps) => {
       });
       return unsubscribeFn;
     };
-  }, []);
+  }, [managerState]);
```

Comment:
Fix openSignInSync: promise will resolve when opened more than once

When an event that can be subscribed was triggered inside the
componentController, the listeners are notified. However the reference
to the list of listeners was never updated, even when a listener was
removed and a new one was added.

This caused the openSingInSync to only resolve once. Which caused the
consumer of openSignInSync to not get notified when the user opened and
closed the sign in modal more than once.

This was extremely painful in the user mgmt component where we need to
fetch the user account settings after connecting to a wallet or other
login method.